src/JestTestRunner.ts

```diff
@@ -122,7 +122,7 @@ export default class JestTestRunner extends EventEmitter implements TestRunner {
   }
 
   private createErrorMessages(result: any[]): string[] {
-    return result.map((r: any) => r.failureMessage);
```

Comment:
I like it because it's explicit. You could of course also just return the `msg` in the `filter`.